pkg/windows/service_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
Do we need to migrate our build tags over to a new format or something?

Reply:
Yes, I believe so. I bumped the version of golanglint-ci to 1.43 and it failed out on the old build tags. With go1.16.9 binary, running a go fmt on these files doesn't fix the build tags. I believe that 1.43.0 is using go1.17 which explains why we are suddenly seeing this.